easybuild/easyconfigs/m/Mesa/Mesa-7.11.2-intel-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,57 @@
+easyblock = 'ConfigureMake'
+
+name = 'Mesa'
+version = '7.11.2'
+
+homepage = 'http://www.mesa3d.org/'
+description = """Mesa is an open-source implementation of the OpenGL specification -
+ a system for rendering interactive 3D graphics."""
+
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'optarch': True}
+
+sources = ['%sLib-%s.tar.gz' % (name, version)]
+source_urls = [
+    'ftp://ftp.freedesktop.org/pub/mesa/%(version)s',
+    'ftp://ftp.freedesktop.org/pub/mesa/older-versions/%(version_major)s.x/%(version)s',
+]
+
+pythonver = '2.7.9'
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+versionsuffix = '-%s-%s' % ('Python', pythonver)
+
+dependencies = [
+    ('flex', '2.5.39', '', ('GCC', '4.9.2')),
+    ('Bison', '3.0.3', '', ('GCC', '4.9.2')),
+    ('makedepend', '1.0.5'),
+    ('Python', pythonver),
+    ('libxml2', '2.9.2', versionsuffix),
+    ('glproto', '1.4.17'),
+    ('libdrm', '2.4.59'),
+    ('libX11', '1.6.2', '-Python-2.7.9'),
+    ('libXt', '1.1.4'),
+    ('libXext', '1.3.3'),
+    ('xproto', '7.0.27'),
+    ('libXdamage', '1.1.4'),
+    ('libXfixes', '5.0.1'),
+]
+
+configopts = " --disable-osmesa --enable-glu --disable-gallium-llvm --disable-gallium-gbm --enable-glx --with-driver=xlib"
+configopts += " --disable-driglx-direct --with-gallium-drivers='' --without-demos --disable-egl"""
+
+# package-config files for os dependencies are in an os specific place
+preconfigopts = ' PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/lib64/pkgconfig/:/usr/share/pkgconfig" '
```

Comment:
```
PKG_CONFIG_PATH

    A colon-separated (on Windows, semicolon-separated) list of directories to search for .pc files. The default 
directory will always be searched after searching the path; the default is libdir/pkgconfig:datadir/pkgconfig where 
libdir is the libdir where pkg-config and datadir is the datadir where pkg-config was installed. 
```

The trouble with these things is that we end up with a bunch of cruft and nobody really knows why it's there (or even if it's still needed).